include/cblox/core/tsdf_submap_collection.h

```diff
@@ -79,10 +90,13 @@ class TsdfSubmapCollection {
   // Gets the tsdf submap associated with the passed ID
   bool getAssociatedTsdfSubMapID(const SubmapID submap_id,
                                  SubmapID *submap_id_ptr) const;
+  bool getTsdfSubmapConstPtrById(const SubmapID submap_id,
```

Comment:
Return pointers by value. Indicate problems by returning a nullptr. See here for an example:

https://github.com/ethz-asl/voxblox/blob/master/voxblox/include/voxblox/core/layer.h#L71

Reply:
Ah yes, that's a nicer solution. Thanks for showing me.